src/raytracer/Raytracer.m

```diff
@@ -56,10 +57,10 @@
 outputPaa = cell(paraCfgInput.numberOfNodes,paraCfgInput.numberOfNodes);
 
 % Input checking
-if paraCfgInput.switchQDGenerator == 1 &&...
+if paraCfgInput.switchDiffuseComponent == 1 &&...
         paraCfgInput.carrierFrequency ~= 60e9
     warning(['Please, note that diffuse scattering model is only ',...
-        'valid for fc=60 GHz'])
+        'valid for fc=60 or 28 GHz'])
```

Comment:
modify if statement accordingly.
more in general this could even be removed: different material libraries consider different frequencies and at the moment this cannot be checked

Reply:
True, I removed this if statement completely